docs/PacketFence_Network_Devices_Configuration_Guide.asciidoc

```diff
@@ -6146,6 +6146,38 @@ Example:
 
 To apply the configuration, restart PacketFence using the following command: service packetfence restart
 
+Ruckus Roles
+^^^^^^^^^^^^
+
+[float]
+Roles Configuration
++++++++++++++++++++
+
+On the Ruckus you can define roles, this roles links all users to the internal WLAN and permits access to all WLANs by default but you can limit their access to certain WLANs.
```

Comment:
Ruckus allows you to define roles. These roles link all users to the internal WLAN and permit access to all WLAN by default. You can still limit access to certain WLAN.